com.unity.multiplayer.mlapi/Runtime/Logging/FileLogger.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+
+namespace MLAPI.Logging
+{
+    /// <summary>
+    /// Helper class for logging
+    /// Saves to a file identified by the process ID of each client
+    /// This is useful when debugging what happens on multiple machines
+    /// ! Only recommended for debugging, not for production !
+    /// </summary>
+    public class FileLogger
+    {
+        public static FileLogger Get() { return instance; }
```

Comment:
You can use the C# accessors & make it on-demand like this:

```suggestion
        public static FileLogger _Instance;
        public static FileLogger Instance
        {
            get
            {
                if (_Instance == null)
                {
                    _Instance = new FileLogger();
                }

                return _Instance;
            }
        }
```

Reply:
minor: `m_Instance` :)

Reply:
:+1: will fix.